src/languages/r.coffee

```diff
@@ -0,0 +1,35 @@
+# Get Atom defaults
+scope = ['text.html']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 1)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
+module.exports = {
+
+  name: "R"
+  namespace: "r"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "R"
+]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "r"
+    "R"
+  ]
+
+  options:
+    indent_size:
```

Comment:
Hi Glavin,

- I installed codo (npm install -g codo)
- I went to atom-beautify folder and executed codo to build docs
- I checked docs/code/class where there is an R.html now.
- I restarted atom in developer mode (atom -d).

->I still do not see indent size setting in Settings - Packages - Development Packages - atom-beautify/settings - R within atom-beautify package.

A screenshot of exactly what I do see:
![screen shot 2016-08-24 at 11 35 05](https://cloud.githubusercontent.com/assets/1298588/17925845/df3417f8-69ee-11e6-93da-ac6a6a7e5e47.png)